test/cmd/lfstest-gitserver.go

```diff
@@ -838,6 +859,13 @@ func (c LocksByCreatedAt) Swap(i, j int)      { c[i], c[j] = c[j], c[i] }
 var lockRe = regexp.MustCompile(`/locks/?$`)
 
 func locksHandler(w http.ResponseWriter, r *http.Request) {
+	repo, err := repoFromLfsUrl(r.URL.Path)
```

Comment:
Nit: we already have this data in the `lfsHandler`, which is what calls this. I would tend to prefer passing that in as a last parameter, though this does make this method less of a "handler" in that it takes more than just a (`http.ResponseWriter`, `*http.Request`).